src/Chainweb/Chainweb/Configuration.hs

```diff
@@ -105,6 +105,7 @@ import qualified Chainweb.Mempool.Mempool as Mempool
 import Chainweb.Mempool.P2pConfig
 import Chainweb.Miner.Config
 import Chainweb.Pact.Types (defaultReorgLimit, defaultModuleCacheLimit, defaultLocalRewindDepthLimit)
+import Chainweb.Pact.Service.Types (RewindLimit(RewindLimit))
```

Comment:
Seems oddly specific, maybe this is a tooling thing? Ordinarily I'd write `RewindLimit(..)`.